citizen_report/report/models.py

```diff
@@ -1,39 +1,20 @@
 from django.db import models
 from django.contrib.auth.models import User
-from django.utils.translation import gettext_lazy as _
 
+from .constants import Category, Status
 
-class Report(models.Model):
-    class Category(models.TextChoices):
-        ROADS_AND_BRIDGES = 'RBR', _('Roads and bridges'),
-        SEWER_AND_WATERWORKS = 'SWS', _('Sewer and waterworks'),
-        POWER_SUPPLY = 'POW', _('Power supply'),
-        GASWORKS = 'GAS', _('Gasworks'),
-        TELECOMUNICATION = 'TEL', _('Telecommunication'),
-        GARBAGE_DISPOSAL = 'GAR', _('Garbage disposal'),
-        CITY_TRANSPORT = 'CTR', _('City transport'),
-        HEALTHCARE = 'HTH', _('Healthcare'),
-        EDUCATION = 'EDU', _('Education'),
-        PUBLIC_SAFETY = 'SAF', _('Public safety')
-
-    class Status(models.TextChoices):
-        FRESH_REPORT = 'FR', _('Fresh report'),
-        CHECKING = 'CH', _('Checking'),
-        VERYFIED = 'VD', _('Veryfied'),
-        APPROVED = 'AP', _('Approved'),
-        WORK_IN_PROGRESS = 'WP', _('Work in progres'),
-        EVALUATION_OF_WORKS = 'EW', _('Evaluation of works'),
-        CLOSED_DONE = 'CD', _('Closed - done'),
 
+class Report(models.Model):
     title = models.CharField(max_length=120)
     text = models.TextField()
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
     image = models.ImageField(upload_to='images/', blank=True)
     category = models.CharField(max_length=3, choices=Category.choices)
-    status = models.CharField(max_length=2, choices=Status.choices, default='FRESH_REPORT')
+    status = models.CharField(max_length=2, choices=Status.choices, default='PN')
+    # TODO: not sure if this was the case - hard coding
```

Comment:
dalej zahardcodowane, Rafał miał to dobrze ogarnięte, podglądnijcie jak to u niego było, uczmy się od siebie też

Generalnie takie przemyślenia są do usunięcia, tym bardziej, że to nie jest TODO